website/pages/docs/connect/proxies/integrate.mdx

```diff
@@ -11,46 +11,130 @@ description: >-
 # Connect Custom Proxy Integration
```

Comment:
The bulk of this was a massaged lift of: https://github.com/hashicorp/consul/pull/8635